public/app/plugins/datasource/loki/result_transformer.ts

```diff
@@ -133,7 +136,9 @@ export function processEntry(
     timeEpochMs,
     timeLocal,
     uniqueLabels,
+    hasAnsi,
     entry: line,
+    raw: hasAnsi ? ansicolor.strip(line) : line,
```

Comment:
If we introduce a `raw` property, it makes more sense for it have the `line` value, and then use the stripped version in the `entry` property.